src/box/lua/key_def.h

```diff
@@ -56,6 +63,71 @@ luaT_push_key_def(struct lua_State *L, const struct key_def *key_def);
 struct key_def *
 luaT_check_key_def(struct lua_State *L, int idx);
 
+/**
+ * Free a key_def from a Lua code.
+ */
+int
+lbox_key_def_gc(struct lua_State *L);
```

Comment:
AFAIU we can make it `static` again (and drop the declaration from the header).

Reply:
Yes, done.